src/averageAges.js

```diff
@@ -35,6 +44,16 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  let womens;
+
+  if (!withChildren) {
+    womens = people.filter(person => person.sex === 'f');
+  } else {
+    womens = people.filter(person => person.sex === 'f' && people.some(child => child.mother === person.name));
+  }
+
+  return womens.reduce((a, x) =>
+    a + (x.died - x.born), 0) / womens.length;
```

Comment:
```suggestion
  return womens.reduce(
  (a, x) => a + (x.died - x.born),
  0
  ) / womens.length;
```
Избегай стрелок в пустоту, нельзя разрывать коллбек, если переносишь, то переноси всю начинку а не часть. Потом, каждый аргумент должен быть с новой строки. Форматируй красиво